grails-app/assets/javascripts/streama/templates/dash.tpl.htm

```diff
@@ -2,7 +2,7 @@
 <div class="dashboard">
   <!--------------------------new releases--------------------------->
   <div ng-if="vm.newReleases.length && !$root.selectedGenre && vm.isDashType('home') && !vm.isDashSectionHidden('new-releases')">
-    <h3>{{'DASHBOARD.NEW_RELEASES' | translate}}</h3>
+    <h3 style="padding-top:1em;">{{'DASHBOARD.NEW_RELEASES' | translate}}</h3>
```

Comment:
Not optimal but the easiest way to only style "new releases". If highlights are enabled the text is too close to the header.